app/assets/javascripts/discourse-common/addon/helpers/html-safe.js

```diff
@@ -1,6 +1,11 @@
+import { helper } from "@ember/component/helper";
 import { htmlSafe } from "@ember/template";
 import { registerUnbound } from "discourse-common/lib/helpers";
 
 registerUnbound("html-safe", function (string) {
   return htmlSafe(string);
 });
+
+export default helper(function ([string]) {
+  return htmlSafe(string);
+});
```

Comment:
With https://github.com/ember-polyfills/ember-functions-as-helper-polyfill, we could just `import { htmlSafe } from "@ember/template";` from the component and use `{{htmlSafe "foo"}}` directly without wrapping it in `helper(...)`

Reply:
Nice! Let's get that addon installed so we can start using this feature in advance of getting to 4.x 👌